modules/dateDisplay.js

```diff
@@ -0,0 +1,7 @@
+const dateContainer = document.querySelector('.date');
+const dateDisplay = () => {
+  const date = new Date();
```

Comment:
- [ ] Great job with the date and time display. Unfortunately, the luxon library is to be used for this task. Take this as a means to learn both methods. Kindly use the luxon library for the date.
- kindly ensure the Package is added as a .js file to the project (in full or minified version)
- then Module is imported `import { DateTime } from "luxon";`

Reply:
Please note that when using luxon library, the linters show a lot of errors, for instance, that the file shall not contain more than one class.